backend/src/test/java/com/bakdata/conquery/integration/IntegrationTests.java

```diff
@@ -160,13 +161,15 @@ private DynamicTest readTest(Resource resource, String name) {
 		try (InputStream in = resource.open()) {
 			JsonIntegrationTest test = new JsonIntegrationTest(in);
 
-			name = test.getTestSpec().getLabel();
+			String testLabel = Optional.ofNullable(test.getTestSpec().getLabel())
+									   // If no label was defined use the filename part before the first dot
+									   .orElse(name.split("\\.", 1)[0]);
```

Comment:
man muss nicht mehr ein label für einen json test setzen

Reply:
Juhu

Reply:
Vlt Ordner+Dateiname?

Reply:
Ordner ist denke ich nicht so wichtig weil die Tests schon in den `dynamicContainers` ein geordnet werden, die die Ordner Struktur abbilden